style.css

```diff
@@ -5,39 +5,37 @@
 
 .borderT {
   background-color: grey;
-  border-top:  1px solid black;
-  border-right:  1px solid black;
-  border-left:  1px solid black;
+  border-top: 1px solid black;
+  border-right: 1px solid black;
+  border-left: 1px solid black;
 }
 
 .borderTW {
-  border-top:  1px solid black;
-  border-right:  1px solid black;
-  border-left:  1px solid black;
+  border-top: 1px solid black;
+  border-right: 1px solid black;
+  border-left: 1px solid black;

```

Comment:
- [ ] **[OPTIONAL]** I noticed that these highlighted CSS properties are being repeated severally in this file. Kindly put these properties in one class and make use of the class anywhere you need these properties. This will make the code clean and prevent repetition.